dashboard/dashboard-user.md

```diff
@@ -54,7 +54,7 @@ TiDB Dashboard 与 TiDB 使用相同的用户权限体系和登录验证方式
     GRANT SYSTEM_VARIABLES_ADMIN ON *.* TO 'dashboardAdmin'@'%';
     ```
 
-- 当所连接的 TiDB 服务器启用了[安全增强模式 (SEM)](/system-variables.md#tidb_enable_enhanced_security) 时，你可以通过执行以下示例 SQL 语句创建一个允许登录 TiDB Dashboard 的 SQL 用户 `dashboardAdmin`：
+- 当所连接的 TiDB 服务器启用了[安全增强模式 (SEM)](/system-variables.md#tidb_enable_enhanced_security) 时，你可以先关闭 SEM，通过执行以下示例 SQL 语句创建一个允许登录 TiDB Dashboard 的 SQL 用户 `dashboardAdmin` 后再重新开启 SEM：
```

Comment:
Why do we need to do this? I cannot understand the reason.

Reply:
I just found after enabling the SEM, the root user can't grant `dashboardAdmin` `RESTRICTED_xxx` privileges, and it reports error:

```mysql
mysql> GRANT RESTRICTED_STATUS_ADMIN ON *.* TO 'dashboardAdmin'@'%';
ERROR 1227 (42000): Access denied; you need (at least one of) the GRANT OPTION privilege(s) for this operation
```

And the root user self has no `RESTRICTED_xxx` privileges.

Then I notice in the permission isolation docs, it says:

> Execute following SQL should disable SEM first, after execute finish, enable SEM again.

Reply:
I guess it is caused by the `root` user does not have sufficient privilege to grant privileges in the SEM mode.

Reply:
```suggestion
- 当所连接的 TiDB 服务器启用了[安全增强模式 (SEM)](/system-variables.md#tidb_enable_enhanced_security) 时，先关闭 SEM，然后执行以下示例 SQL 语句创建一个允许登录 TiDB Dashboard 的 SQL 用户 `dashboardAdmin` ，创建完成后，再重新开启 SEM：
```